x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/rule_execution_logic/eql/trial_license_complete_tier/eql_alert_suppression.ts

```diff
@@ -61,6 +69,10 @@ export default ({ getService }: FtrProviderContext) => {
     log,
   });
 
+  const partitionSequenceBuildingBlocks = (
+    alerts: Array<SearchHit<DetectionAlert & RiskEnrichmentFields>>
+  ) => partition(alerts, (alert) => alert?._source?.['kibana.alert.building_block_type'] == null);
```

Comment:
```suggestion
  ) => partition(alerts, (alert) => alert?._source?.['kibana.alert.group.index'] == null);
```
If the EQL rule is a building block rule, the shell alert will also be a building block. The `group.id` and `group.index` fields are the definitive identifiers for sequence alerts - `group.id` is shared across all the subAlerts and the shell alert, and `group.index` is populated on each subAlert and defines which event in the sequence the alert corresponds to. The shell alert does not use `group.index`.